taxonomy/app/schemas/taxon.py

```diff
@@ -39,3 +39,18 @@ class TaxonResponseSchema(TaxonInputSchema):
     class Config:
         allow_population_by_field_name = True
         orm_mode = True
+
+
+class ParentsConcatenateResponseSchema(BaseModel):
+    taxon_id: str = Field(..., example="my_taxon_id")
+    taxon_name: str = Field(..., example="taxon_name")
+    parent_ids_concat: Optional[str] = Field(
+        ..., example="parent_id_1.parent_id_2"
+    )
+    parent_names_concat: Optional[str] = Field(
+        ..., example="parent_name_1.parent_name_2"
+    )
+
+    class Config:
+        allow_population_by_field_name = True
+        orm_mode = True
```

Comment:
Do you this model with ORM?
If not - there's no point of using this attribute

Reply:
agree, removed both